sympy/physics/control/lti.py

```diff
@@ -76,6 +77,54 @@ def bilinear(tf, sample_per):
         return num_coefs, den_coefs
 
 
+def backward_diff(tf, sample_per):
+
+        """
+
+        Returns falling coefficients of H(z) from numerator and denominator.
+        Where H(z) is the corresponding discretized transfer function,
+        discretized with the backward difference transform method.
+        H(z) is obtained from the continuous transfer function H(s)
+        by substituting s(z) =  (z-1)/(T*z) into H(s), where T is the
+        sample period.
+        H(z) corresponds to the difference equation of the by backward difference scheme.
+        Coefficients are falling, i.e. H(z) = (az+b)/(cz+d) is returned
+        as [a, b], [c, d].
+
+        Examples
+
+        ========
+
+        >>> from sympy.physics.control.lti import TransferFunction, backward_diff
+        >>> from sympy.abc import s, L, R, T
+        >>> tf = TransferFunction(1, s*L + R, s)
+        >>> numZ, denZ = backward_diff(tf, T)
+        >>> numZ
+        [T, 0]
+        >>> denZ
+        [L + R*T, -L]
+        """
+
+        z = Symbol('z') # discrete variable z
```

Comment:
This should be passed in as a parameter to the function.

Reply:
@asmeurer , thank you for feedback.  Should z be an Input parameter, even if it is used only as dummy variable, i.e. If it is not part of the output?

Reply:
@asmeurer , @hanspi42, 

in `bilinear` and in `backward_diff` the dummy discrete variable "z" is now the internal var symbol, i.e. the variable of the corresponding transfer function.

Then, there is no neeed anymore of defining a new Symbol (z = Symbol('z'))

Since `bilinear` and in `backward_diff` return only coefficients this OK from my point of view.

Reply:
@asmeurer , @hanspi42, can I resolve this conversation? Thank you